### DISCUSSION TITLE
/me (analytics) crashes

### DISCUSSION THREAD

**Discussion by ludviglindblom** [cssbattle/forum#267] [Bugs reports]
When I visit the [analytics page](https://cssbattle.dev/me) it shows up for a brief moment, displaying the loading animation the the text **Chrunching your stats...** and then it crashes.

The screen just says 

> Application error: a client-side exception has occurred (see the browser console for more information).

And in the console it reads as in the attached image:

<img width="644" alt="Screenshot 2022-06-09 at 15 42 09" src="https://user-images.githubusercontent.com/445328/172862464-6c7a51be-2917-43b7-96d8-630638ba62db.png">

**Comment by chinchang** [comment]
Sorry about that...locked targets were messing the logic there 😅. Fixed now!

**Reply by ludviglindblom** [reply]
Awesome. Thanks for the quick fix.
